packages/react-components/src/molecules/__tests__/UsersList.test.tsx

```diff
@@ -54,6 +54,53 @@ it('does not link external users', () => {
   expect(getByText(/John Doe/).closest('a')).toBeNull();
 });
 
+describe('alumni badge', () => {
+  it('shows alumni badge for internal and alumni users', () => {
+    const { getByText } = render(
+      <UsersList
+        max={1}
+        users={Array.from({ length: 1 }).map(
+          () =>
+            ({
+              ...createUserResponse(),
+              displayName: 'John Doe',
+              alumniSinceDate: new Date(2021, 6, 12, 14, 32).toISOString(),
+            } as ExternalAuthorResponse),
```

Comment:
Why "as ExternalAuthorResponse" when the user is a CRN one?
And why use Array.from instead of ` users={[
        {
          ...createUserResponse(),
          displayName: 'John',
          id: '42',
         alumniSinceDate: new Date(etc.),
        },
      ]}`